test/test_alexa_app_creation.js

```diff
@@ -0,0 +1,30 @@
+/*jshint expr: true*/
+"use strict";
+var chai = require("chai");
+var chaiAsPromised = require("chai-as-promised");
+chai.use(chaiAsPromised);
+var expect = chai.expect;
+chai.config.includeStack = true;
+
+describe("Alexa", function () {
+    var Alexa = require("../index");
+
+    describe("app", function () {
+        var testApp;
```

Comment:
You don't need this variable here, or assign anything to the call inside the try/catch below.